test/bootstrap/cloudtrace.sh

```diff
@@ -0,0 +1,4 @@
+#!/bin/bash
+
+# Enable the CloudTrace API
+gcloud services enable cloudtrace.googleapis.com
```

Comment:
Well, it is not enough to generate a good test, right? You need to create this trace too to have it in the output of the tool. Could you provide an example on how to do it via bash?